test/nix/test.nix

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file can be empty as we do not actually run anything in the test. Is just to check if the command to run it is correct.

Reply:
Thanks, updated :)